### PR TITLE
1.0.0 changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
- Added SS and icons
- Redrafted content
- Added missing navigation buttons

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -1,6 +1,71 @@
...
+
+    - Click **I will complete later** to collapse that task's row
+
+    - The task action buttons remain active and appear when the row is expanded
+
+- **Not Applicable**: Select this button if a task does not apply to your FortiSOAR environment.
+
+    ![](./res/setup-guide-na.png)
+
+    - Click **Not Applicable** to display that task with a green circular check. A **Reset Task Options** button appears in its place.
```
Comment (cs-avit-suryawanshi) [inline docs/usage.md]: When we click "Not Applicable" , it displays white strikeout tick.

File: docs/setup.md [hunk 2]
```diff
@@ -1,11 +1,28 @@
-| [Home](../README.md) |
-|--------------------------------------------|
-
 # Installation
 
+1. To install a solution pack, click **Content Hub** > **Discover**.
```
Comment (cs-suyog-jinturkar) [inline docs/setup.md]: SP?

File: docs/setup.md [hunk 3]
```diff
@@ -1,11 +1,28 @@
...
+2. From the list of solution pack that appears, search **Setup Guide**.
```
Comment (cs-suyog-jinturkar) [inline docs/setup.md]: SP?

File: docs/setup.md [hunk 4]
```diff
@@ -1,11 +1,28 @@
-| [Home](../README.md) |
-|--------------------------------------------|
-
 # Installation
```
Comment (cs-suyog-jinturkar) [inline docs/setup.md:4]: we can add a note saying Setup guide automatically installed as part of SFSP v3 and later.

File: docs/setup.md [hunk 5]
```diff
@@ -1,11 +1,28 @@
...
 
+To view the FortiSOAR setup guide, you must be assigned a role with the following minimum permissions:
+
+1. `Read` and `Update` permissions on the **Application** module
+2. `Read` and `Update` permissions on the **Security** module
 
 # Configuration
 
-This wizard doesn't need any configuration settings.
+The widget is a part of [**SOAR Framework**](https://github.com/fortinet-fortisoar/solution-pack-soar-framework) solution pack `v3.0.0` and does not require any standalone configuration.
```
Comment (cs-suyog-jinturkar) [inline docs/setup.md]: This note should be present in installation section too

**Review by cs-avit-suryawanshi — APPROVED** [review]

File: docs/usage.md [hunk 6]
```diff
@@ -1,6 +1,71 @@
...
+
+    - Click **I will complete later** to collapse that task's row
+
+    - The task action buttons remain active and appear when the row is expanded
+
+- **Not Applicable**: Select this button if a task does not apply to your FortiSOAR environment.
+
+    ![](./res/setup-guide-na.png)
+
+    - Click **Not Applicable** to display that task with a struck-out, circular check ![icon-green-check](./res/icon-not-applicable.svg). A **Reset Task Options** button appears in its place.
```
Comment (cs-avit-suryawanshi) [inline docs/usage.md]: "icon-not-applicable" should be shown instead of "icon-green-check"

**Review by cs-avit-suryawanshi — APPROVED** [review]

File: docs/usage.md (hunk 6 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md]: You mean the `alt=` text. Will change. Thanks. Check in dark mode as well.

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**